src/components/WorkQueueIconText.vue

```diff
@@ -1,8 +1,8 @@
 <template>
```

Comment:
This component got a lot more complicated in this PR - maybe we should break the work pool stuff into its own icon-text component instead to simplify and make the work pool name a required prop

Reply:
But we don't know if we're going to get workPoolName or workPoolId, we cannot make it required. And depending on what we get, we then do different subscriptions to get data we need. For some reason, some models like deployment return pool name, and some like old workQueues return id, and they cannot be updated because it breaks something on the backend. Also, there's no point in keeping `WorkQueueIconText` since it's not used anywhere else